### PR TITLE
fix readiness probe

### PR DESCRIPTION
**Pull request by vegichan**
The probe returned too early and could not be correctly used for readiness checks.

### CODE REVIEW

File: build/etc/charts/nuodb-helm/templates/sts-admin.yaml [hunk 1]
```diff
@@ -85,7 +85,7 @@ spec:
           initialDelaySeconds: 30
           periodSeconds: 15
           exec:
-            command: [ "nuodocker", "check", "servers" ]
+            command: [ "nuodocker", "check", "servers", "--check-connected", "--check-active", "--check-leader" ]
```
Comment (thebithead) [inline build/etc/charts/nuodb-helm/templates/sts-admin.yaml]: Will that command work for all supported versions of NuoDB?

Comment (vegichan) [reply build/etc/charts/nuodb-helm/templates/sts-admin.yaml]: what are those?

Comment (thebithead) [reply build/etc/charts/nuodb-helm/templates/sts-admin.yaml]: Presumably, any version of NuoDB that support Docker.  Let me ask the question in a different way: what version of NuoDB has support for the command line switches that you added?

Comment (vegichan) [reply build/etc/charts/nuodb-helm/templates/sts-admin.yaml]: There is only one currently released version of NuoDB that supports New Admin and that is 4.0. Hence the operator and the helm charts can only use NuoDB 4.0 and onwards.

These flags have been around for a long time. http://doc.nuodb.com/Latest/Default.htm#NuoDB-Command-Commands.htm?Highlight=nuocmd

Comment (thebithead) [reply build/etc/charts/nuodb-helm/templates/sts-admin.yaml]: This change looks fine

However it works outside the control of the NuoDB Operator.  I think that later on, the NuoDB Operator should play a more active role in determining readiness and liveness and taking corrective action (self-heal, auto-tune, etc).

Comment (vegichan) [reply build/etc/charts/nuodb-helm/templates/sts-admin.yaml]: thanks!

**Pull request merged**